content/en/docs/contributing/release-process.md

```diff
@@ -261,7 +266,8 @@ page if a step is missing or if it is outdated.
         ```sh
         # Must be run from the cert-manger folder.
         export GITHUB_TOKEN=*your-token*
-        go install k8s.io/release/cmd/release-notes@v0.7.0
+        git fetch origin $BRANCH:$BRANCH
+        export START_SHA="$(git rev-list --reverse --ancestry-path $START_TAG..$BRANCH | head -1)"
```

Comment:
This won't work when releasing v1.8.0. Let's imagine I am about to release v1.8.0 and that I want to generate the release notes. Since v1.7.0 does not belong to `master`, the command:

```sh
git rev-list --reverse --ancestry-path v1.7.0..master | head -1
```

returns nothing.

To work around the fact that v1.7.0 does not belong to master, we could consider the "base" commit between master and v1.7.0:

```sh
git rev-list --reverse --ancestry-path $(git merge-base v1.7.0 master)..master | head -1
```

I will update the release process page.